src/pages/docs/infrastructure/deployment-targets/kubernetes/kubernetes-agent/troubleshooting.md

```diff
@@ -44,6 +44,14 @@ If the Agent install command fails with a timeout error, it could be that:
 - (if using the NFS storage solution) The NFS CSI driver has not been installed
 - (if using a custom Storage Class) the Storage Class name doesn't match
 
+#### 404 error when setting up the NFS Pod.
```

Comment:
@APErebus `The 404 error when setting up the NFS pod should also just a troubleshooting node`
did you mean to put it here under `Installation Issues`?